rpcs3/Emu/Cell/Modules/cellSearch.cpp

```diff
@@ -76,13 +138,39 @@ error_code cellSearchInitialize(CellSearchMode mode, u32 container, vm::ptr<Cell
 
 error_code cellSearchFinalize()
 {
-	cellSearch.warning("cellSearchFinalize()");
+	cellSearch.todo("cellSearchFinalize()");
 
 	const auto search = g_fxo->get<search_info>();
 
+	switch(search->state)
+	{
+		case SearchState::IDLE:
+			break;
+		case SearchState::NOT_INITIALIZED:
+			return CELL_SEARCH_ERROR_NOT_INITIALIZED;
+		case SearchState::FINALIZING:
+			return CELL_SEARCH_ERROR_FINALIZING;
+		case SearchState::IN_PROGRESS:
+		case SearchState::INITIALIZING:
+		case SearchState::CANCELING:
+			return CELL_SEARCH_ERROR_BUSY;
+		default:
+			return CELL_SEARCH_ERROR_GENERIC;
+	}
+
+	search->state = SearchState::FINALIZING;
+
+	// Umount hard links
```

Comment:
What's Umount

Reply:
https://www.thegeekdiary.com/how-to-mount-and-umount-a-file-system-in-linux/

I guess both umount and unmount are accepted terminology.